src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
+    private static final String SPECIFIED_CHARACTER = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+        File file = new File(fileName);
+        try {
+            List<String> listOfWords = Files.readAllLines(new File(fileName).toPath());
+            return wordsFromListLowerCase(listOfWords);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read file " + fileName, e);
+        }
+    }
+
+    private String[] wordsFromListLowerCase(List<String> listOfWords) {
+        StringBuilder textBuilder = new StringBuilder();
+
+        for (String word : listOfWords) {
+            textBuilder.append(word.toLowerCase()).append(System.lineSeparator());
+            if (textBuilder.length() <= 0) {
```

Comment:
```suggestion
            if (textBuilder.length() == 0) {
```
It never will be less than zero